routes/api-admin.php

```diff
@@ -49,3 +49,6 @@
 Route::post('/User::create', [UserController::class, 'create']);
 Route::post('/User::setEmail', [UserController::class, 'setEmail'])->withoutMiddleware([AdminCheck::class]);
 Route::post('/User::getEmail', [UserController::class, 'getEmail'])->withoutMiddleware([AdminCheck::class]);
+Route::post('/User::getCurrent', [UserController::class, 'getCurrent'])->withoutMiddleware([AdminCheck::class]);
```

Comment:
Is the name of the route specified by some standard or can it be changed? I have not yet reviewed what `getCurrent` does but IMHO it should tell what it gets, e.g. `getCurrentSomething`. I assume token? Why not simply `getToken`?

Reply:
After I have been reading on, I assume `getAuthenticatedUser` is what has been meant?